src/libraries/Microsoft.PowerFx.Core/IR/Nodes/BinaryOpKind.cs

```diff
@@ -75,6 +75,7 @@ internal enum BinaryOpKind
         LeqTime,
         GtTime,
         GeqTime,
+        DynamicGetField,
```

Comment:
>DynamicGetField, [](http://example.com/codeflow?start=8&length=16)

This is great. Now hosts that don't impl it will just get a NotImpl exception from their unhandled switch.